neoload/commands/__init__.py

```diff
@@ -1,2 +1,2 @@
 __all__ = ["login", "logout", "logs_url", "project", "run", "status", "stop", "test_results", "test_settings",
-           "validate", "wait", "zones"]
+           "validate", "wait", "zones", "report"]
```

Comment:
Do we need to also add in the list : "config", "docker", "fastfail", "workspaces" ?